pdr_backend/lake/ohlcv_data_factory.py

```diff
@@ -85,8 +86,31 @@ def get_mergedohlcv_df(self) -> pl.DataFrame:
         asyncio.run(self._update_rawohlcv_files(fin_ut))
         logger.info("Update all rawohlcv files: done")
         rawohlcv_dfs = self._load_rawohlcv_files(fin_ut)
-        mergedohlcv_df = merge_rawohlcv_dfs(rawohlcv_dfs)
+        for feed in self.ss.feeds:
+            columns = [
+                "timestamp",
+                "open",
+                "high",
+                "low",
+                "close",
+                "volume",
+                "Cumulative Dollar Value",
+                "Cumulative Ticks",
+            ]
+            df = rawohlcv_dfs[str(feed.exchange)][str(feed.pair)]
+            if feed.threshold is not None:
+                logger.info(f"Get {feed.threshold.prefix} bars for %s", feed)
+                bars = []
+                df_pandas = df.to_pandas()
+                bars = self._get_threshold_bars(
+                    df_pandas, feed.threshold.prefix, feed.threshold.threshold
```

Comment:
you can just pass the feed.threshold to the function.
inside the function, get these two attributes